src/main/java/core/basesyntax/Plane.java

```diff
@@ -7,5 +7,100 @@
  */
 
 public class Plane {
+    private String name;
+    private String model;
+    private int maxSpeed;
+    private double maxHeight;
+    private int maxSeats;
 
+    private Plane(PlaneBuilder builder) {
+        this.name = builder.name;
+        this.model = builder.model;
+        this.maxSpeed = builder.maxSpeed;
+        this.maxHeight = builder.maxHeight;
+        this.maxSeats = builder.maxSeats;
+    }
+
+    public Plane(String name, String model, int maxSpeed, double maxHeight, int maxSeats) {
+        this.name = name;
+        this.model = model;
+        this.maxSpeed = maxSpeed;
+        this.maxHeight = maxHeight;
+        this.maxSeats = maxSeats;
+    }
+
+    // Getters
+    public String getName() {
+        return name;
+    }
+
+    public String getModel() {
+        return model;
+    }
+
+    public int getMaxSpeed() {
+        return maxSpeed;
+    }
+
+    public int getMaxSeats() {
+        return maxSeats;
+    }
+
+    public double getMaxHeight() {
+        return maxHeight;
+    }
+
+    public class PlaneBuilder {
+        private String name;
+        private String model;
+        private int maxSpeed;
+        private double maxHeight;
+        private int maxSeats;
+
+        //Getters
+        public double getMaxHeight() {
+            return maxHeight;
+        }
+
+        public int getMaxSeats() {
+            return maxSeats;
+        }
+
+        public int getMaxSpeed() {
+            return maxSpeed;
+        }
+
+        public String getModel() {
+            return model;
+        }
+
+        public String getName() {
+            return name;
+        }
+
+        // Setters
+        public void setMaxHeight(double maxHeight) {
```

Comment:
```suggestion
        public PlaneBuilder setMaxHeight(double maxHeight) {
```

Reply:
do you mean that I need to change return type of every setters from void to PlaneBuilder?